src/control/states/DraggingState.ts

```diff
@@ -40,8 +40,12 @@ class DraggingState extends State {
 
     const camera = flicking.camera;
     const prevPosition = camera.position;
+    const position = axesEvent.pos[AXES.POSITION_KEY];
```

Comment:
Is the onChange code of DraggingState and AnimatingState the same, is it difficult to commonize?

Reply:
Right, looks like they're identical. I should deal with that.